brasilio_auth/templates/brasilio_auth/login.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+{% if form.errors %}
+<p>Your username and password didn't match. Please try again.</p>
```

Comment:
I'm not sure if the best approach now is to have everything in English than translate, since the other views are in Portuguese. What do you think?

Reply:
Note: the file `brasilio_auth/templates/brasilio_auth/password_reset_complete.html` is in Portuguese, so there's no consistency.

Reply:
This was my bad! Copy 'n' paste errors =/

I'll update it